src/oops/SOLID/DependencyInversionPrinciple/before/serielizer/EmployeeSerializer.java

```diff
@@ -0,0 +1,7 @@
+package oops.SOLID.DependencyInversionPrinciple.before.serielizer;
+
+import oops.SOLID.DependencyInversionPrinciple.before.employees.Employee;
+
+public interface EmployeeSerializer {
+    public String serialize(Employee employee);
```

Comment:
interface methods are public by default, you don't have to specify it explicitly.

Reply:
Thanks for that. Didn't remember this fact explicitly.